nestkernel/nest_time.h

```diff
@@ -387,7 +387,13 @@ class Time
   bool
   is_neg_inf() const
   {
-    return tics == LIM_NEG_INF.tics;
+    return tics <= LIM_NEG_INF.tics;
```

Comment:
Is this supposed to be called after computations that stay in range, but go beyond the limit? I ask to check -- why should tics ever be legally < LIM_NEG_INF or > LIM_POS_INF?